Makefile

```diff
@@ -46,7 +46,7 @@ DEPS_CFLAGS +=	-pthread
 
 OBJ_DIR =	$(TOP)/obj
 
-CTFCONVERT =	/opt/ctf/bin/ctfconvert
+CTFCONVERT =	/bin/true
```

Comment:
This looks a bit weird. binder.git is running its own copy of ctfconvert on the balancer binary built in this repository. Could we just remove all the ctfconvert machinery here then, or are we leaving this enabled just for documentation purposes? If that's the case, it'd be good to document that, perhaps something along the lines of

```
# When built standalone, we may not have a local copy of the ctfconvert tool, so we set
# CTFCONVERT to /bin/true here. When balancer is built as a git submodule of binder.git,
# we run ctfconvert on the balancer binary separately.
```

[update: oh, I see this is touched on in the README, perhaps the Makefile comment is worth adding
then ]